qa/pull-tester/rpc-tests.py

```diff
@@ -29,26 +30,22 @@
 import tempfile
 import re
 
-sys.path.append("qa/pull-tester/")
-from tests_config import *
-
 BOLD = ("","")
 if os.name == 'posix':
     # primitive formatting on supported
     # terminal via ANSI escape sequences:
     BOLD = ('\033[0m', '\033[1m')
 
-RPC_TESTS_DIR = SRCDIR + '/qa/rpc-tests/'
+# Read config generated by configure.
+config = configparser.ConfigParser()
+config.read_file(open(os.path.dirname(__file__) + "/tests_config.ini"))
+
+ENABLE_WALLET = config["components"]["ENABLE_WALLET"] == "True"
```

Comment:
Maybe just use the config object directly below instead of defining these variables, since they are only used once. This will cut down on the number of global variables in this script, and make it clearer from the context where the settings are used where the settings come from.